src/test/java/io/prometheus/cloudwatch/CloudWatchCollectorTest.java

```diff
@@ -297,15 +297,6 @@ public void testGetDimensionsUsesNextToken() throws Exception {
   }
 
   @Test
-  public void testGetMonitoringEndpoint() throws Exception {
```

Comment:
Why remove this test?

Reply:
I initially removed it because it's testing internal implementation, but also realized it's not being tested elsewhere in the test suite.  Re-added in 13a857c.